hack/e2e.go

```diff
@@ -262,6 +262,8 @@ func TearDown() bool {
 
 // Up brings an e2e cluster up, recreating it if one is already running.
 func Up() bool {
+	// force having batch/v2alpha1 always on for e2e tests
+	os.Setenv("KUBE_RUNTIME_CONFIG", "batch/v2alpha1=true")
```

Comment:
I get that this is DRY'er than setting a default in all of the `clusters/*/config-foo.sh` scripts but I don't know if it's a pattern we want to extend as more features get added?